portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,32 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Aymar!");
+    /*These lines used to return a Hello Aymar message*/
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hello Aymar!");
+    ArrayList<String> msg = new ArrayList<String>();
+    msg.add("First message.");
+    msg.add("This is the second string.");
+    msg.add("Last string.");
```

Comment:
Nit: Consider using [Arrays.asList](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-)

Reply:
Changed the message container to a class to be able to use gson more efficiently